game-app/game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/MainPanel.java

```diff
@@ -36,6 +38,7 @@
 public class MainPanel extends JPanel {
   private static final long serialVersionUID = -5548760379892913464L;
   private static final Dimension initialSize = new Dimension(900, 780);
+  private static MainPanel instance;
```

Comment:
I see why you added this member, but having such a static member leads to bad habits.
I once tried to get rid of all the root frame member, but that rabbit hole was just too deep.
So if there's a way to stick the the existing static members in main frame and not introduce any more I'd prefer that, but if it's too complicated then it's not that important.